paper-badge.html

```diff
@@ -153,11 +153,21 @@
         icon: {
           type: String,
           value: ''
+        },
+
+        /**
+         * Treated as the hidden attribute.
+         */
+        hidden: {
+          type: Boolean,
+          reflectToAttribute: true
         }
       },
 
       attached: function() {
-        this._updateTarget();
+        requestAnimationFrame(function() {
```

Comment:
took me forever to find out why again, this is because `this.offsetParent` is not defined on `attached` in polymer 2.x. I've added a comment to this section